manifests/virt-handler.yaml.in

```diff
@@ -29,6 +30,8 @@ spec:
         volumeMounts:
         - name: libvirt-runtime
           mountPath: /var/run/libvirt
+        - name: sockets
```

Comment:
I was thinking each type data would be in a separate sub directory.

**share directory**
/var/run/kubevirt

**top level sub directory for all ephemeral data associated with a VM called 'myvm' in namespace 'default**
/var/run/kubevirt/default/myvm

**'myvm's socket subdirectory**
/var/run/kubevirt/default/myvm/sockets/sock

**'myvm's cloud init subdirectory**
/var/run/kubevirt/default/myvm/cloudInit/noCloud.iso

virt-launcher associated with some VM called 'myvm' would only have access to the 'myvm' subdirectory. Virt-handler and libvirt would have access to the whole /var/run/kubevirt/ directory tree.

By having all ephemeral data associated with a VM tied to a single top level directory, it makes garbage collection a little easier. We wouldn't have to keep up with all the different components needing to share data between components.  We'd just know to delete a top level sub directory if the VM is no longer around. 

Also, as new features come around that need to share things between components related to a domain, all they have to do is drop their data into the VM's subdirectory. There would be some guarantees about how that data is cleaned up that everyone gets for free.